core/src/main/java/nl/nn/adapterframework/http/HttpSender.java

```diff
@@ -81,9 +81,10 @@
 import nl.nn.adapterframework.util.XmlUtils;
 
 /**
- * Sender for the HTTP protocol using GET, POST, PUT or DELETE.
- *
- * Returns HTTP status code as forward name to SenderPipe.
+ * Sender for the HTTP protocol using GET, POST, PUT or DELETE. By default, any response code outside the 2xx or 3xx range
```

Comment:
Een beetje een zeurpuntje maar als je dan toch bezig bent is het wel handig te verwijzen naar alle mogelijke http methodes.
```suggestion
 * Sender for the HTTP protocol using {@link nl.nn.adapterframework.http.HttpSenderBase.HttpMethod HttpMethod}. By default, any response code outside the 2xx or 3xx range
```